satellite/audit/service.go

```diff
@@ -63,13 +84,33 @@ func (service *Service) Run(ctx context.Context) (err error) {
 	defer mon.Task()(&ctx)(&err)
 	service.log.Info("Audit cron is starting up")
 
-	return service.Loop.Run(ctx, func(ctx context.Context) error {
-		err := service.process(ctx)
-		if err != nil {
-			service.log.Error("process", zap.Error(err))
-		}
-		return nil
+	group, ctx := errgroup.WithContext(ctx)
+
+	//group.Go(func() error {
+	//	return service.Loop.Run(ctx, func(ctx context.Context) error {
+	//		defer mon.Task()(&ctx)(&err)
+	//		err := service.process(ctx)
+	//		if err != nil {
+	//			service.log.Error("process", zap.Error(err))
+	//		}
+	//		return nil
+	//	})
+	//})
+
+	group.Go(func() error {
```

Comment:
Is the group necessary here?

Also sync.Cycle has a `Start` that takes group as an argument.

Reply:
the group should be necessary since now we have two different audit loops running even though the new one using the metainfoloop is only creating reservoirs and not using them for selection yet.

Reply:
What I mean is that the loops have the necessary function you don't need to use `group.Go`.

Reply:
so I should be doing it like this with both loops?
```
		service.Loop.Start(ctx, group, func(ctx context.Context) (err error) {
			defer mon.Task()(&ctx)(&err)
			err = service.process(ctx)
			if err != nil {
				service.log.Error("process", zap.Error(err))
			}
			return nil
		})
```

Reply:
Yeah.

Reply:
thanks!